securedrop/tests/test_i18n.py

```diff
@@ -173,6 +174,33 @@ def test_verify_default_locale_en_us_if_not_defined_in_config(self):
         finally:
             config.DEFAULT_LOCALE = DEFAULT_LOCALE
 
+    def test_locale_to_rfc_5646(self):
+        assert i18n.locale_to_rfc_5646('en') == 'en'
+        assert i18n.locale_to_rfc_5646('en-US') == 'en'
+        assert i18n.locale_to_rfc_5646('en_US') == 'en'
+        assert i18n.locale_to_rfc_5646('en-us') == 'en'
+        assert i18n.locale_to_rfc_5646('zh-hant') == 'zh-Hant'
+
+    def test_html_lang_correct(self):
+        app = journalist.app.test_client()
+        resp = app.get('/', follow_redirects=True)
+        html = resp.data.decode('utf-8')
+        assert re.compile('<html .*lang="en".*>').search(html), html
+
+        # TODO add check for a non-English language on the journalist app
```

Comment:
Can you remove these TODOs and add a @checklist item in #2230 to remind us we should add testing for `lang="fr"` when we merge in the French translation?